crypto/keys/pem/pem_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	. "github.com/google/trillian/crypto/keys/pem"
 	ktestonly "github.com/google/trillian/crypto/keys/testonly"
 	"github.com/google/trillian/testonly"
+
+	_ "github.com/golang/glog"
```

Comment:
why this change?

Reply:
Some of those tests seem to be run by hand by the GCB scripts (in addition to being run via `go test`), and get passed some glog flags, which are not recognised without this.

This was previously pulled indirectly, somehow (not obvious to me in this PR, but there's been much code deleted), it was found via GCB failures and going through the error logs.